app/dashboard/management/commands/activity_report.py

```diff
@@ -0,0 +1,133 @@
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+'''
+
+import argparse
+import csv
+import datetime
+import re
+import StringIO
+from itertools import imap
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from app.utils import itermerge
+from dashboard.models import Bounty, Tip
+from marketing.mails import send_mail
+
+DATE_FORMAT = '%Y/%m/%d'
+def valid_date(v):
+    try:
+        return datetime.datetime.strptime(v, DATE_FORMAT)
+    except ValueError:
+        raise argparse.ArgumentTypeError('%s is not a date in YYYY/MM/DD format' % v)
+
+GITHUB_REPO_PATTERN = re.compile('github.com/[\w-]+/([\w-]+)')
+
+
+class Command(BaseCommand):
+
+    help = 'emails activity report of tips and bounties to settings.CONTACT_EMAIL'
+
+    def add_arguments(self, parser):
+        parser.add_argument('start_date', type=valid_date, help='Start of date range (inclusive) in YYYY/MM/DD format for activities to be collected')
+        parser.add_argument('end_date', type=valid_date, help='End of date range (inclusive) in YYYY/MM/DD format for activities to be collected')
+
+    def extract_github_repo(self, url):
+        match = GITHUB_REPO_PATTERN.search(url)
+        if not match:
+            self.stdout.write(self.style.WARNING('WARNING: malformed github url: %s, using value as is' % url))
+            return url
+
+        return match.groups()[0]
+
+    def format_bounty(self, bounty):
+        return {
+            'type': 'bounty',
+            'created_on': bounty.web3_created,
+            'last_activity': bounty.modified_on,
+            'amount': bounty.value_in_token,
+            'denomination': bounty.token_name,
+            'amount_eth': bounty.value_in_eth,
+            'amount_usdt': bounty.value_in_usdt,
+            'from_address': bounty.bounty_owner_address,
+            'claimee_address': bounty.claimeee_address,
+            'repo': self.extract_github_repo(bounty.github_url),
+            'from_username': bounty.bounty_owner_github_username or '',
+            'claimee_github_username': bounty.claimee_github_username or '',
+            'status': bounty.status,
+            'comments': ''
+        }
+
+    def format_tip(self, tip):
+        return {
+            'type': 'tip',
+            'created_on': tip.created_on,
+            'last_activity': tip.modified_on,
+            'amount': tip.amount,
```

Comment:
can you use `tip.get_natural_value()` here?  b960da1fc0d6b05a9960d028fcc759f4a93595d8 is now on master, and it adds this method